web/src/components/TitledField.tsx

```diff
@@ -0,0 +1,15 @@
+import {ErrorMessage, Field} from "formik";
+import {styles} from "../styles/tailwindGlobals";
+
+interface TitledFieldProps {
+  name: string;
+  label: string;
+}
+
+export const TitledField = ({name, label}: TitledFieldProps) => (
+    <div>
+      <label className={styles.label} htmlFor={name}>{label}</label>
```

Comment:
IIUC. The `htmlFor` field value should be the `id` of the `input` element. That means that we should need to create a unique `id` value for every `input` field. We could use [@accessible/use-id](https://github.com/accessible-ui/use-id) for this.

Reply:
nice thanks